src/components/widgets-common/top-buttons/top-buttons.directive.coffee

```diff
@@ -16,6 +16,20 @@ module.controller('CommonTopButtonsCtrl', ($scope, $rootScope, $log, ImpacWidget
 
   $scope.hasInfo = ->
     w && w.content? && w.content.info? && w.content.info.length > 0
+
+  $scope.isCsvExportable = ->
+    templates = _.filter(ImpacDashboardsSvc.getWidgetsTemplates(), {name: w.name})
```

Comment:
`w.name` is an editable field, I suggest using `w.endpoint` to find the correct template, as it's the most consistent identifier for templates. 

Also, rather than using `_.filter`, then `templates[0]`, can you just use `template =  _.find(..)` ?